Homework38/myproject/myproject/settings.py

```diff
@@ -1,4 +1,4 @@
-from users.models import MyModel
+from users.models import User
```

Comment:
It is impossible to load the User model until `users` application is not loaded yet.
This line is redundant and could be simply removed.